modules/Movecraft/src/main/java/net/countercraft/movecraft/warfare/siege/Siege.java

```diff
@@ -7,25 +7,31 @@
 import java.util.concurrent.atomic.AtomicReference;
 
 public class Siege {
-    @NotNull
+    @NotNull private final List<Integer> daysOfWeek;
     private final List<String> craftsToWin, commandsOnStart, commandsOnLose, commandsOnWin;
```

Comment:
`private final List<String> craftsToWin, commandsOnStart, commandsOnLose, commandsOnWin;` still has its annotation removed.

Reply:
Should be fixed now.